app/views/tables.html

```diff
@@ -10,7 +10,7 @@
         <div class="col-sm-6 col-xs-12">
           <div class="panel panel-info">
             <div class="panel-heading">
-              <p class="announcement-heading"><b>{{ table_label }}</b> <span ng-show="isParted" class="small">(partitioned)<span></p>
+              <p class="announcement-heading"><b>{{ (nr_of_tables == 0 && ('TABLE.TITLE' | translate) + ' (' + nr_of_tables + ')') || table_label }}</b> <span ng-show="isParted" class="small">(partitioned)<span></p>
```

Comment:
a && b || c or, in Python: myvar  = a and b or c where a, b, and c are expressions. a is the boolean condition that should apply, and if it does, b also has to be true (in the JS/Python sense) to be returned to the caller (this is a language feature. In Java it would only return 'true'). If a evaluates to false, short-circuit evaluation means that if the first part (a) is already false, just skip ahead and return whatever is the second part of the OR, since this will be the outcome. 

ehm. makes sense? I don't like the solution either, maybe there's something better.